modules/product/src/main/java/com/opengamma/strata/product/bond/FixedCouponBond.java

```diff
@@ -161,10 +161,28 @@ private void validate() {
         "The ex-coupon period is measured from the payment date, thus the days must be non-positive");
   }
 
-  //-------------------------------------------------------------------------
+  /**
+   * Resolves fixed coupon bond using specified reference data.
+   * @param refData  the reference data to use when resolving
+   * @return the resolved instance
+   * @throws ReferenceDataNotFoundException if an identifier cannot be resolved in the reference data
+   * @throws RuntimeException if unable to resolve due to an invalid definition
+   */
   @Override
   public ResolvedFixedCouponBond resolve(ReferenceData refData) {
-    Schedule adjustedSchedule = accrualSchedule.createSchedule(refData);
+    return resolve(refData, false);
```

Comment:
My feeling is that we can just change this to use true instead of false. This avoids adding the extra method. It seems that combining periods is the normal thing most users would expect.

Reply:
sure, let me update the MR

Reply:
@jodastephen can you please check the latest commit. I have removed the method and put true as default